src/main/java/Todo.java

```diff
@@ -0,0 +1,13 @@
+public class Todo extends Task{
```

Comment:
Name of class might fit the coding standards better if it was changed to "ToDo" instead? 
(Class/enum names must be nouns and written in PascalCase E.g Line, AudioSystem).
```suggestion
public class ToDo extends Task{
```